src/native/api.c

```diff
@@ -127,6 +127,15 @@ const char* llhttp_method_name(llhttp_method_t method) {
 }
 
 
+void llhttp_set_lenient(llhttp_t* parser, int enabled) {
+  if (enabled) {
+    parser->flags |= F_LENIENT;
+  } else if (parser->flags & F_LENIENT) {
+    parser->flags ^= F_LENIENT;
+  }
+}
```

Comment:
Shorter/branchless way of doing the same (not that it really matters but, you know, it looks neat):
```c
parser->flags ^= F_LENIENT & parser->flags;
parser->flags ^= F_LENIENT * !!enabled;
```

Reply:
Good point. While we are here, am I right that `&= ~F_LENIENT` would not work reliably here? (out of curiosity)

Reply:
It does actually but two statements with a common prefix is aesthetically pleasing to me. :-)

Reply:
Thanks!

Reply:
I decided to go with `~F_LENIENT` as it is less tricky and explicit.